crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -284,3 +324,66 @@ pub async fn main(cfg: &mut Config) -> Result<()> {
     cfg.save()?;
     Ok(())
 }
+
+pub async fn main(no_daemon: bool, cfg: &mut Config) -> Result<()> {
+    if no_daemon {
+        run_sandbox(cfg).await?;
+    } else {
+        let path = sandbox_daemon_log_path()?;
+        let stdout_file = OpenOptions::new()
+            .create(true)
+            .write(true)
+            .truncate(true)
+            .open(path.clone())?;
+
+        let mut child = Command::new(std::env::current_exe()?)
+            .args(["sandbox", "start", "--no-daemon"])
+            .stdout(Stdio::from(stdout_file))
+            .spawn()?;
```

Comment:
While this is essentially what forking a process is, it doesn't adhere to the standard way of creating a daemon. 

Did you look into using `daemonize`?

Reply:
yes, I've spend significantly more time on trying to make it work through daemonize compared to this version. However, I've encountered multiple issues with daemonize - for example it isn't very compatible with LocalSet and how we run jstz-node. It also had some networking and configuration issues.

My guess is that it probably possible to make it somehow work through daemonize - however it would take more time to get it work and doesn't really functionally change it compared to this version, therefore I would suggest we keep it as is now, focus on more high priority features before the doogfooding release and then we can look at it again later if you really want.

Reply:
Just spent 10 mins playing around with this -- it isn't an issue with `LocalSet` or `jstz-node`, but the tokio runtime. 

Namely daemonize uses fork under the hood, but the tokio runtime breaks after forking (in the child process). To support this we'd need to carefully re-write bits of the sandbox to initialize the runtime _after_ forking.